lib/spack/spack/build_systems/cmake.py

```diff
@@ -83,14 +83,21 @@ def build_type(self):
         return 'RelWithDebInfo'
 
     @property
-    def root_cmakelists_dir(self):
-        """Returns the location of the root CMakeLists.txt
+    def cmakelists_directory(self):
+        """The relative path to the directory containing CMakeLists.txt
```

Comment:
I see. I renamed it to be more in line with what `AutotoolsPackage` calls it, but I could undo this change. Don't some Autotools projects also have multiple configure scripts? That might just be for vendored dependencies, but I know OpenMPI has a few.